htdocs/langs/ar_IQ/mails.lang

```diff
@@ -137,6 +137,7 @@ MailSendSetupIs=Configuration of email sending has been setup to '%s'. This mode
 MailSendSetupIs2=You must first go, with an admin account, into menu %sHome - Setup - EMails%s to change parameter <strong>'%s'</strong> to use mode '%s'. With this mode, you can enter setup of the SMTP server provided by your Internet Service Provider and use Mass emailing feature.
 MailSendSetupIs3=If you have any questions on how to setup your SMTP server, you can ask to %s.
 YouCanAlsoUseSupervisorKeyword=You can also add the keyword <strong>__SUPERVISOREMAIL__</strong> to have email being sent to the supervisor of user (works only if an email is defined for this supervisor)
+YouCanAlsoUseCreatorKeyword=You can also add the keyword <strong>__SUPERVISOREMAIL__</strong> to have email being sent to the object author (works only if an email is defined for this supervisor)
```

Comment:
No need to fill the language file, only en_US must be done. the other will be done automatically by transifex translation synchronization.